core/src/main/java/io/questdb/griffin/OperatorExpression.java

```diff
@@ -54,6 +54,7 @@
         add(new OperatorExpression("and", 11, true, BINARY, false));
         add(new OperatorExpression("or", 11, true, BINARY, false));
         add(new OperatorExpression("not", 11, true, UNARY, false));
+        add(new OperatorExpression("like", 7, true, BINARY));
```

Comment:
'like' must not be a symbol (see 'and', 'not' etc.)

The reason is that symbols break up words, for example 'dislike' will be tokenized as 'dis' + 'like' if 'like' was a symbol. When 'like' is not a symbol - 'dislike' will remain whole and 'like' will only be tokenized as such when it is separated with whitespace, e.g. ` x like '%a'`

Reply:
@bluestreak01 understood